src/screens/Ticketing/Tickets/Tabs.tsx

```diff
@@ -71,8 +75,26 @@ export const BuyTickets: React.FC<Props> = ({navigation}) => {
     }
   };
 
+  const enableTicketingOverlay = () => {
+    appContext.resetTicketing();
+  };
+
   return (
     <View style={styles.container}>
+      <View style={{padding: theme.spacings.medium}}>
+        <MessageBox>
+          <ThemeText type="body__primary" color="primary_1">
+            Enkelte begrensninger gjelder
```

Comment:
Dette må inn i text-filene for språkstøtte.

Reply:
:facepalm: